extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -65,3 +103,57 @@ export const renderPageAttachmentUI = (pageEl, attachmentEl) => {
   textEl.textContent = openLabel;
   return openAttachmentEl;
 };
+
+/**
+ * Renders inline page attachment UI.
+ * @param {!Window} win
+ * @param {!Element} pageEl
+ * @param {!Element} attachmentEl
+ * @return {!Element}
+ */
+const renderInlinePageAttachmentUI = (win, pageEl, attachmentEl) => {
+  const openAttachmentEl = buildOpenInlineAttachmentElement(pageEl);
+
+  const openLabelAttr = attachmentEl.getAttribute('data-cta-text');
+  if (openLabelAttr) {
+    const textEl = win.document.createElement('span');
+    textEl.classList.add('i-amphtml-story-inline-page-attachment-label');
+    const openLabel = openLabelAttr && openLabelAttr.trim();
+    textEl.textContent = openLabel;
+    openAttachmentEl.appendChild(textEl);
+    // Override descriptive text on page-attachment button.
+    openAttachmentEl.setAttribute('aria-label', openLabel);
```

Comment:
This is a regression, we're losing i18n here.
It should be the same as above

```
const openLabelAttr = attachmentEl.getAttribute('data-cta-text');
  const openLabel =
    (openLabelAttr && openLabelAttr.trim()) ||
    getLocalizationService(pageEl).getLocalizedString(
      LocalizedStringId.AMP_STORY_PAGE_ATTACHMENT_OPEN_LABEL
    );
```

Reply:
Something new in the new UI is that there is an option for no text with the button, whereas previously we would default to this `AMP_STORY_PAGE_ATTACHMENT_OPEN_LABEL` text if none was provided. So because of that, the way I've changed this now is that the `aria-label` is now either:

1) the provided text from the publisher, matching the label on the CTA button.
2) the default `AMP_STORY_PAGE_ATTACHMENT_OPEN_LABEL` text, which will be the accessibility text when there's no visible label provided.

Reply:
We could not override the i18n string but that assumes that the publisher's text is decorative. 

I wonder if it would work to prepend the publisher specified string to the i18n label.

Reply:
Ok so the solution we've gone with now is using the default i18n "Swipe up" text for the component and the aria-label unless the publisher specifies text or specifies 'none'.